Detections/AzureActivity/NewResourceGroupsDeployedTo.yaml

```diff
@@ -1,32 +1,35 @@
 ﻿id: 9fb57e58-3ed8-4b89-afcf-c8e786508b1c
-name: Common deployed resources
+name: Suspicious Resource deployment
 description: |
-  this query looks for common deployed resources (resource name and resource groups). Alerts is
-  generated for new ResourceGroups first seen associated with deployment operations.
+  'Identifies when a rare Resource and ResourceGroup deployment occurs by a previously unseen Caller.'
```

Comment:
nit:
I wrote it in another PR. 
I think the decription should be wrapped with ' character only when : character use in the description, otherwise it could be used but not necessary.